SimCalorimetry/HGCalAssociatorProducers/plugins/BuildFile.xml

```diff
@@ -0,0 +1,7 @@
+<use name="FWCore/Framework"/>
+<use name="FWCore/PluginManager"/>
+<use name="FWCore/ParameterSet"/>
+<use name="SimDataFormats/Associations"/>
+<use name="RecoLocalCalo/HGCalRecAlgos"/>
```

Comment:
@civanch @slava77 is Reco dependence in a Sim package allowed?

Reply:
@civanch @slava77 ping

Reply:
A producer/plugin is an end-point in dependencies.
So, there's likely not much harm.
Some reduction in dependencies may be possible, but for now this isn't changing what's already done elsewhere (there was a dependence of Validation on  HGCalRecAlgos)